implementations/c/config/vault/ockam_vault_hw_cfg.h

```diff
@@ -0,0 +1,69 @@
+/**
+ ********************************************************************************************************
+ * @file        ockam_vault_hw_cfg.h
+ * @author      Mark Mulrooney <mark@ockam.io>
+ * @copyright   Copyright (c) 2019, Ockam Inc.
+ * @brief   
+ ********************************************************************************************************
+ */
+
+#ifndef OCKAM_VAULT_HW_CFG_H_
+#define OCKAM_VAULT_HW_CFG_H_
+
+/*
+ ********************************************************************************************************
+ *                                             INCLUDE FILES                                            *
+ ********************************************************************************************************
+ */
+
```

Comment:
do we really need this header? doesn't a series of `#include` statements convey the same thing.

Reply:
It ensures consistency of include location across all files.